test/common/stats/stats_matcher_test.cc

```diff
@@ -0,0 +1,246 @@
+#include "envoy/config/metrics/v2/stats.pb.h"
+
+#include "common/stats/stats_matcher_impl.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+using testing::IsFalse;
+using testing::IsTrue;
+
+namespace Envoy {
+namespace Stats {
+
+class StatsMatcherTest : public testing::Test {
+public:
```

Comment:
The usual pattern I've seen to is to mark everything in the the Test class as protected.

Reply:
Done.